src/App.test.js

```diff
@@ -6,4 +6,4 @@ test('renders learn react link', () => {
   const { getByText } = render(<App />);
   const linkElement = getByText(/learn react/i);
   expect(linkElement).toBeInTheDocument();
-});
+});
```

Comment:
Same comment as earlier - this doesn't seem like an intentional change

Reply:
Done